clap_man/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "clap_man"
+version = "3.0.0-rc.4"
```

Comment:
Let's drop this version down to pre-1.0 so we make it clear that this is still in early access.

Reply:
Done.